infrastructure/aws/ecs.tf

```diff
@@ -67,13 +55,14 @@ resource "aws_route53_record" "type_a_record" {
 
 
 module "django-app" {
-  create_listener    = false
-  create_networking  = false
+#   registry_arn = aws_service_discovery_service.service_discovery.arn
+  create_listener    = true
+  create_networking  = true
   memory             = 4096
   cpu                = 2048
   source             = "../../../i-ai-core-infrastructure//modules/ecs"
   project_name       = "redbox-django-app"
-  image_tag          = var.image_tag
+  image_tag          = "0a08cb1d50315e20258d835f904af19b26a26a78"
```

Comment:
Is hard-coding this the best way?

Reply:
nope. this is very much WIP. this PR only really exists right now to build the images so i can deploy them

Reply:
Fair dos.